src/classes/issues-processor.ts

```diff
@@ -556,8 +557,13 @@ export class IssuesProcessor {
         (issue: Readonly<OctokitIssue>): Issue => new Issue(this.options, issue)
       );
     } catch (error) {
-      this._logger.error(`Get issues for repo error: ${error.message}`);
-      return Promise.resolve([]);
+      const rethrow = Error(
+        `Getting issues was blocked by the error: ${error.message}`
+      );
+      rethrow.stack = `${error.stack.split('\n').slice(0, 2).join('\n')}\n${
+        error.stack
+      }`;
```

Comment:
Do we really need the stack? In my understanding, the error message is enough to understand what went wrong.